backend/inference.py

```
@@ -1,369 +1,186 @@
+
+    out["findingType"] = "real_media"
+    if origin_score is None:
+        return out
+    if origin_score > 0.5:
+        out["confidence"] = 50
+        out["insufficientEvidence"] = True
+        out["reason"] = "Face and full-frame detectors disagree; result is inconclusive."
+        out["disputed"] = True
+    else:
+        out["confidence"] = min(
+            int(base.get("confidence", 50)), int(round((1.0 - origin_score) * 100))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)